web/app/themes/mitlib-parent/css/scss/global.scss

```diff
@@ -1,7 +1,8 @@
 @import "modules/all";
 @import "main";
 @import "responsive";
-@import "../../libs/fontawesome/css/font-awesome";
-@import "../../libs/fontawesome-MITLibraries/style";
+@import "https://cdn.dev1.mitlibrary.net/files/fontawesome6.6.0/css/fontawesome.css";
```

Comment:
Let's promote fonteawesome to the prod CDN and update this before merging. I'll reach out on slack to schedule time to show you that process.